doc/start/hardware-recommendations.rst

```diff
@@ -54,55 +54,60 @@ Monitors and managers (ceph-mon and ceph-mgr)
 Monitor and manager daemon memory usage generally scales with the size of the
 cluster.  Note that at boot-time and during topology changes and recovery these
 daemons will need more RAM than they do during steady-state operation, so plan
-for peak usage.  For very small clusters, 32 GB suffices.  For
-clusters of up to, say, 300 OSDs go with 64GB.  For clusters built with (or
-which will grow to) even more OSDS you should provision
-128GB.  You may also want to consider tuning settings like ``mon_osd_cache_size``
-or ``rocksdb_cache_size`` after careful research.
+for peak usage. For very small clusters, 32 GB suffices. For clusters of up to,
+say, 300 OSDs go with 64GB. For clusters built with (or which will grow to)
+even more OSDs you should provision 128GB. You may also want to consider
+tuning the following settings:
+
+* :confval:`mon_osd_cache_size`
+* :confval:`rocksdb_cache_size`
+
 
 Metadata servers (ceph-mds)
 ---------------------------
 
 The metadata daemon memory utilization depends on how much memory its cache is
 configured to consume.  We recommend 1 GB as a minimum for most systems.  See
-``mds_cache_memory``.
+:confval:`mds_cache_memory_limit`.
 
 
 Memory
 ======
 
 Bluestore uses its own memory to cache data rather than relying on the
 operating system page cache.  In bluestore you can adjust the amount of memory
-the OSD attempts to consume with the ``osd_memory_target`` configuration
+the OSD attempts to consume with the :confval:`osd_memory_target` configuration
 option.
 
-- Setting the osd_memory_target below 2GB is typically not recommended (it may
-  fail to keep the memory that low and may also cause extremely slow performance.
+- Setting the :confval:`osd_memory_target` below 2GB is typically not
+  recommended (it may fail to keep the memory that low and may also cause
+  extremely slow performance.
 
 - Setting the memory target between 2GB and 4GB typically works but may result
-  in degraded performance as metadata may be read from disk during IO unless the
+  in degraded performance: metadata may be read from disk during IO unless the
```

Comment:
I've been trying to replace `disk` with `drive` or `storage`as I go along, but that's not worth going out of your way.

Reply:
Or, in this sentence, maybe just "metadata may be read during IO", skipping the whole explicit metaphor entirely.